sdk/core/azure-core/test/ut/environmentLogLevelListener_test.cpp

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <gtest/gtest.h>
+#include <private/environment_log_level_listener.hpp>
+
+using Azure::Core::Diagnostics::Logger;
+using Azure::Core::Diagnostics::_detail::EnvironmentLogLevelListener;
+
+namespace {
+void SetVariable(std::string const& value)
+{
+#if defined(_MSC_VER)
+  static_cast<void>(_putenv(("AZURE_LOG_LEVEL=" + value).c_str()));
+#else
+  static_cast<void>(setenv(std::string("AZURE_LOG_LEVEL").c_str(), value.c_str(), 1));
+#endif
+}
+} // namespace
+
+TEST(EnvironmentLogLevelListener, LogLevelDefault)
+{
+  EnvironmentLogLevelListener::SetInitialized(false);
+  SetVariable("unknown");
+  auto level = EnvironmentLogLevelListener::GetLogLevel(Logger::Level::Verbose);
+  EXPECT_EQ(level, Logger::Level::Verbose);
+
+  EnvironmentLogLevelListener::SetInitialized(false);
+  SetVariable("");
+  level = EnvironmentLogLevelListener::GetLogLevel(Logger::Level::Verbose);
+  EXPECT_EQ(level, Logger::Level::Verbose);
```

Comment:
surprised to see the default as `verbose`. I would expected to be `no-logging` or just `error`.  I guess I missed that decision in the past and now it would be a breaking change to update. :(

Reply:
Must be consistent with Java.